eos-update-flatpak-repos

```diff
@@ -1031,6 +1031,39 @@ FLATPAKS_TO_MIGRATE = [
         'new-branch': 'stable',
         'old-origin': 'eos-apps',
         'new-origin': 'flathub'
+    },
+    # migrate endlessnetwork apps to flathub (T25928)
+    {
+        'name': 'co.com.btg.Conductor',
```

Comment:
https://github.com/flathub/co.com.btg.Conductor doesn't exist. These all need to specify the new name.